cmd/snap/cmd_quota_test.go

```diff
@@ -285,18 +284,21 @@ func (s *quotaSuite) TestSetQuotaCpuHappy(c *check.C) {
 		}
 	}`
 	routes := map[string]http.HandlerFunc{
-		"/v2/quotas": makeFakeQuotaPostHandler(
+		"/v2/quotas": s.makeFakeQuotaPostHandler(
 			c,
 			fakeHandlerOpts,
 		),
-		"/v2/quotas/foo": makeFakeGetQuotaGroupHandler(c, fmt.Sprintf(getJsonTemplate, 1000)),
+		"/v2/quotas/foo": s.makeFakeGetQuotaGroupHandler(c, fmt.Sprintf(getJsonTemplate, 1000)),
 		"/v2/changes/42": makeChangesHandler(c),
 	}
 	s.RedirectClientToTestServer(dispatchFakeHandlers(c, routes))
 
 	// ensure that --cpu still works with cgroup version 1
 	_, err := main.Parser(main.Client()).ParseArgs([]string{"set-quota", "--cpu=2x50%", "foo"})
-	c.Assert(err, check.IsNil)
+	c.Check(err, check.IsNil)
+	c.Check(s.quotaGetGroupsHandlerCalls, check.Equals, 0)
```

Comment:
(nitpick) I personally would leave this check (`c.Check(s.quotaGetGroupsHandlerCalls, check.Equals, 0)`) out because the handler is not registered in this test (AFAICT) so there is no possible way this could be not equal 0? Same for the other tests. But maybe there is a good reason to do it or I misunderstand in which case just ignore me :)

Reply:
Really good point! I've updated this now